api/serializers/enrollment_test.py

```diff
@@ -45,8 +52,43 @@ class Meta:
         fields = (
             "student",
             "answers",
-            "resulting_level",
         )
 
     # TODO Think about some validation, e.g. answers should be unique for each question
     #  and should be in options for this question
+
+
+class EnrollmentTestResultLevelSerializer(serializers.ModelSerializer[EnrollmentTestResult]):
+    """A serializer used to get level of language based on how many answers are correct."""
+
+    resulting_level = serializers.SerializerMethodField()
+
+    class Meta:
+        model = EnrollmentTestResult
+        fields = ["answers", "resulting_level"]
```

Comment:
Just a small optimisation: if something is not subject to change, it's better to use immutable data structure (tuple in this case)

Reply:
Done (with mine and yours)